src/howitz/endpoints.py

```diff
@@ -63,7 +63,8 @@ def auth_handler(username, password):
                 session["errors"] = {}
                 session["not_connected_counter"] = 0
                 return user
-    return None
+        else:
+            raise Forbidden('Not authenticated')
```

Comment:
Because you also changed `authenticate_user` to raise an HTTP exception instead of returning None, this is redundant. Pick one.

If you pick that the exception is only raised in `authenticate_user` you no longer need the `if user:`-block, so dedent.